src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,107 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final double GROW_MULTIPLYER = 1.5;
+    private static final int DEFAUL_SIZE = 10;
+    private int size = 0;
+    private T[] elements;
+
+    public ArrayList() {
+        this.elements = (T[]) new Object[DEFAUL_SIZE];
```

Comment:
this is redundant, read about where we need it https://stackoverflow.com/a/2411283